docs/ref/middleware.txt

```diff
@@ -128,7 +128,8 @@ Conditional GET middleware
 Handles conditional GET operations. If the response has a ``ETag`` or
 ``Last-Modified`` header, and the request has ``If-None-Match`` or
 ``If-Modified-Since``, the response is replaced by an
-:class:`~django.http.HttpResponseNotModified`.
+:class:`~django.http.HttpResponseNotModified`. If there is no ``ETag``
+the middleware tries to calculate and set one.
```

Comment:
Add a `.. versionchanged:: 1.10` annotation as described in [documenting new features](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features).